internal/cmd/argus/scrape-config/describe/describe.go

```diff
@@ -0,0 +1,181 @@
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	jobNameArg = "JOB_NAME"
+
+	instanceIdFlag = "instance-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	JobName    string
+	InstanceId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("describe %s", jobNameArg),
+		Short: "Shows details of a scrape configuration from an Argus instance",
+		Long:  "Shows details of a scrape configuration from an Argus instance.",
+		Args:  args.SingleArg(jobNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of a scrape configuration with name "my-config" from Argus instance "xxx"`,
+				"$ stackit argus scrape-config describe my-config --instance-id xxx"),
+			examples.NewExample(
+				`Get details of a scrape configuration with name "my-config" from Argus instance "xxx" in a table format`,
+				"$ stackit argus scrape-config describe my-config --output-format pretty"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("read scrape configuration: %w", err)
+			}
+
+			return outputResult(p, model.OutputFormat, resp.Data)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), instanceIdFlag, "Instance ID")
+
+	err := flags.MarkFlagsRequired(cmd, instanceIdFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	jobName := inputArgs[0]
+
+	globalFlags := globalflags.Parse(cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	return &inputModel{
+		GlobalFlagModel: globalFlags,
+		JobName:         jobName,
+		InstanceId:      flags.FlagToStringValue(cmd, instanceIdFlag),
+	}, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *argus.APIClient) argus.ApiGetScrapeConfigRequest {
+	req := apiClient.GetScrapeConfig(ctx, model.InstanceId, model.JobName, model.ProjectId)
+	return req
+}
+
+func outputResult(p *print.Printer, outputFormat string, config *argus.Job) error {
+	switch outputFormat {
+	case globalflags.PrettyOutputFormat:
+
+		saml2Enabled := "Enabled"
+		if config.Params != nil {
+			saml2 := (*config.Params)["saml2"]
+			if len(saml2) > 0 && saml2[0] == "disabled" {
```

Comment:
what are the other items in the list used for? Is this consistent with the Portal?

Reply:
This is consistent with the Portal, its how I got this information in the first place. I am not aware of any other items using this Params field, but the API also doesn't specify this